ee/modules/110-istio/hooks/ensure_crds_istio.go

```diff
@@ -52,5 +53,9 @@ func ensureCRDs(input *go_hook.HookInput, dc dependency.Container) error {
 	sort.Sort(semver.Collection(semvers))
 
 	CRDversionToInstall := fmt.Sprintf("%d.%d", semvers[len(semvers)-1].Major(), semvers[len(semvers)-1].Minor())
+
+	if os.Getenv("D8_IS_TESTS_ENVIRONMENT") != "" {
+		return ensure_crds.EnsureCRDsHandler("/deckhouse/ee/modules/110-istio/crds/istio/"+CRDversionToInstall+"/*.yaml")(input, dc)
+	}
```

Comment:
use variable please like an another modules

Reply:
I've refactored this condition in https://github.com/deckhouse/deckhouse/pull/3740/commits/76e1162876c282fbd869bd1fae39cbbdfcadbc17.